web.js

```diff
@@ -441,6 +455,17 @@ module.exports = function (options) {
         done()
       })
     }
+
+    function changeToHapi (path) {
+      var tokens = path.split('/')
+
+      for (var i in tokens) {
+        if (tokens[i].indexOf(':') === 0) {
```

Comment:
No, protocol is added by server implementation and depends how it is Express/Hapi initialized. 

Path is the for /api/service